pipelines/build-deploy-test-CI-WL.yml

```diff
@@ -0,0 +1,2436 @@
+      KUBERNETES_CLUSTER: ((ci-kubernetes-cluster-name))
+      KUBERNETES_DEPLOYMENT_NAME: eventlatencymonitor
+      KUBERNETES_SELECTOR: app=eventlatencymonitor
+      KUBERNETES_FILE_PATH: kubernetes-repo/optional
+      KUBERNETES_FILE_PREFIX: eventlatencymonitor-dev
+      WAIT_UNTIL_AVAILABLE_TIMEOUT: 200s
+    input_mapping: {
+      docker-image-resource: toolbox-docker-image-gcr,
+      kubernetes-repo: census-rm-kubernetes-optional-repo}
+
+- name: "CI Deploy Rabbit Monitor"
+  serial: true
+  serial_groups: [ci-deploy-infrastructure,
+                  ci-acceptance-tests]
+  plan:
+  - get: census-rm-terraform
+    trigger: true
+    passed: ["CI Deploy Infrastructure"]
+  - get: census-rm-kubernetes-optional-repo
+    trigger: true
+  - get: toolbox-master
+    trigger: true
+    passed: ["Build Toolbox Latest"]
+  - get: toolbox-docker-image-gcr
+    params:
+      skip_download: true
+  - get: census-rm-deploy
+  - task: apply-deployment
+    file: census-rm-deploy/tasks/kubectl-apply-deployment.yml
+    on_failure: *slack_failure_alert_ci
+    params:
+      SERVICE_ACCOUNT_JSON: ((gcp.service_account_json))
+      GCP_PROJECT_NAME: ((ci-gcp-project-name))
+      KUBERNETES_CLUSTER: ((ci-kubernetes-cluster-name))
+      KUBERNETES_DEPLOYMENT_NAME: rabbitmonitor
+      KUBERNETES_SELECTOR: app=rabbitmonitor
+      KUBERNETES_FILE_PATH: kubernetes-repo/optional
+      KUBERNETES_FILE_PREFIX: census-rm-toolbox
+      WAIT_UNTIL_AVAILABLE_TIMEOUT: 200s
+    input_mapping: {
+      docker-image-resource: toolbox-docker-image-gcr,
+      kubernetes-repo: census-rm-kubernetes-optional-repo}
+
+- name: "CI Deploy Regional Counts"
+  serial: true
+  serial_groups: [ci-deploy-infrastructure,
+                  ci-acceptance-tests]
+  plan:
+  - get: census-rm-terraform
+    trigger: true
+    passed: ["CI Deploy Infrastructure"]
+  - get: census-rm-kubernetes-optional-repo
+    trigger: true
+  - get: toolbox-master
+    trigger: true
+    passed: ["Build Toolbox Latest"]
+  - get: toolbox-docker-image-gcr
+    params:
+      skip_download: true
+  - get: census-rm-deploy
+  - task: apply-deployment
+    file: census-rm-deploy/tasks/kubectl-apply-deployment.yml
+    on_failure: *slack_failure_alert_ci
+    params:
+      SERVICE_ACCOUNT_JSON: ((gcp.service_account_json))
+      GCP_PROJECT_NAME: ((ci-gcp-project-name))
+      KUBERNETES_CLUSTER: ((ci-kubernetes-cluster-name))
+      KUBERNETES_DEPLOYMENT_NAME: regionalcounts
+      KUBERNETES_SELECTOR: app=regionalcounts
+      KUBERNETES_FILE_PATH: kubernetes-repo/optional
+      KUBERNETES_FILE_PREFIX: regional-counts-dev
+      WAIT_UNTIL_AVAILABLE_TIMEOUT: 200s
+    input_mapping: {
+      docker-image-resource: toolbox-docker-image-gcr,
+      kubernetes-repo: census-rm-kubernetes-optional-repo}
+
+
+
+- name: "CI Acceptance Tests"
+  serial: true
+  serial_groups: [ci-deploy-infrastructure,
+                  ci-acceptance-tests,
+                  docker-builds]
```

Comment:
Acceptance tests are now blocked by docker image builds which should prevent test runs from kicking off halfway through a set of builds as we're seeing with the current pattern.